examples/6field-simple/elm_6f.cxx

```diff
@@ -290,7 +290,7 @@ const Field2D N0tanh(BoutReal n0_height, BoutReal n0_ave, BoutReal n0_width,
   mesh->get(Grid_NX, "nx");
   mesh->get(Jysep, "jyseps1_1");
   Grid_NXlimit = n0_bottom_x * Grid_NX;
-  output.write("Jysep1_1 = %i   Grid number = %e\n", int(Jysep), Grid_NX);
+  output.write("Jysep1_1 = {:d}   Grid number = {:e}\n", int(Jysep), Grid_NX);
```

Comment:
Could you split the PR into one with manual changes, and one with automatic changes?

I assume that would significantly simplifying this review.

Reply:
I don't think so, as the PR with the manual changes would be broken.